server/utils/cookieless_utils.ts

```diff
@@ -186,3 +187,16 @@ export async function generateEmbedTokens(
     throw error
   }
 }
+
+export function isValidConfig(res: Response) {
```

Comment:
I was thinking that we would also validate config for acquire and generate. Having said that, that validation would be different so this is fine as is. I would rename to isValidAuthConfig ready for adding additional validation for the other two endpoints